BootloaderCommonPkg/Include/Guid/OsBootOptionGuid.h

```diff
@@ -186,6 +186,8 @@ typedef union {
   ///
   ///
   CONTAINER_IMAGE      ContainerImage;
+  /// RAM raw address
+  UINT32               RamAddress;
```

Comment:
It will create a corner error in LoadBootImages (PayloadPkg/OsLoader/LoadImage.c) for the case "BootImage[Index].LbaImage.Valid == 1", because RamAddress and LbaImage are two fields of the union structure. 

For other fields, it is unlikely to have BootImage[Index].LbaImage.Valid == 1, because the first byte of FileName and FileImage will not be 0x1. However, it is possible for the first byte of RamAddress to be 0x1 (although it will not aligned memory).

Reply:
Updated